src/Microsoft.OData.Core/Json/JsonLightInstanceAnnotationWriter.cs

```diff
@@ -1,236 +1,237 @@
-﻿//---------------------------------------------------------------------
-// <copyright file="JsonLightInstanceAnnotationWriter.cs" company="Microsoft">

```

Comment:
I (now) only see the single change to support writing enum-valued annotations.

---
In reply to: [123297140](https://github.com/OData/odata.net/pull/828#discussion_r123297140) [](ancestors = 123297140)